sdk/resourcemanager/azure-resourcemanager-appplatform/src/main/java/com/azure/resourcemanager/appplatform/models/AppResourceProperties.java

```diff
@@ -141,17 +159,6 @@ public String fqdn() {
         return this.fqdn;
     }
 
-    /**
-     * Set the fqdn property: Fully qualified dns Name.
-     *
-     * @param fqdn the fqdn value to set.
-     * @return the AppResourceProperties object itself.
-     */
-    public AppResourceProperties withFqdn(String fqdn) {
-        this.fqdn = fqdn;
-        return this;
-    }
-
     /**
```

Comment:
revapi.skip here. fqdn has changed to readOnly. We haven't expose this model from our public interface except `innerModel()`.

Reply:
According to service, it's a swagger fix. This property is never allowed to change by user.
https://github.com/Azure/sdk-release-request/issues/4225#issuecomment-1605867112